www/apps/resources/app/recipes/digital-products/examples/standard/page.mdx

```diff
@@ -1817,79 +1817,81 @@ In a later step, you’ll add an API route to allow customers to view and downlo
 
 ---
 
-## Step 12: Handle the Digital Product Order Event
+## Step 12: Fulfill Digital Order Workflow
 
-In this step, you'll create a subscriber that listens to the `digital_product_order.created` event and sends a customer an email with the digital products they purchased.
+In this step, you'll create a workflow that fulfills a digital order. Later, you'll execute this workflow in a subscriber.
 
-Create the file `digital-product/src/subscribers/handle-digital-order.ts` with the following content:
+The workflow has the following steps:
 
-export const subscriberHighlight = [
-  ["20", "notificationModuleService", "Resolve the Notification Module's service to use it later to send a notification."],
-  ["22", "fileModuleService", "Resolve the File Module's service to use it later to retrieve a media's URL."],
-  ["26", "query", "Run the query to retrieve the digital product order."]
-]
+1. Retrieve the digital product order's details. For this, you'll use the `useRemoteQueryStep` imported from `@medusajs/core-flows`.
+2. Send a notification to the customer with the digital products to download.
+3. Create a shipment for the fulfillment to indicate that the item is fulfilled. For this step, you'll use from the `@medusajs/core-flows` package the `createOrderShipmentWorkflow` as a step.
```

Comment:
thought: I think we should consider removing the third step here. Digital products are not shipped and should ideally be created with a `requires_shipping: false` option, which can be used in the storefront to build the appropriate checkout flow. The item processing flow for digital products should end after fulfillment.

Reply:
@olivermrbl ah makes sense, where would `requires_shipping: false` be passed? when the shipping method is set on the cart?

Reply:
I just checked and that's the property of an inventory item, but digital products don't have inventory items (unless that's the better approach?)

Reply:
> but digital products don't have inventory items

Hmm, I guess that depends on the case. For example, you might have a limited amount of licenses. Conversely, it is also very likely that you have an unlimited amount of licenses 😄 

For the recipe, my take would be that we should not have the shipment step. This is likely also how we will build the digital products starter (when we get around to it).

Reply:
Makes sense! Will remove it then